lib/bootstrap_form/helpers/bootstrap.rb

```diff
@@ -46,8 +46,9 @@ def errors_on(name, options={})
         return unless error?(name)
 
         hide_attribute_name = options[:hide_attribute_name] || false
+        custom_class_name   = options[:custom_class_name] || false
```

Comment:
I think the `_name` isn't really needed here, but that's not a strong opinion, so if you prefer to leave it, please do so.

Reply:
Yes, its better without `_name`